.github/workflows/pull-requests.yml

```diff
@@ -79,13 +79,16 @@ jobs:
   test--unit:
     needs: [build-head]
     runs-on: [self-hosted, ci]
+    strategy:
+      matrix:
+        test-type: [unit, slow]
```

Comment:
This looks like a sneaky extra change haha - what does this achieve?

Reply:
not sure i follow - otherwise the `slow` tests i wrote won't run?